polls/admin.py

```diff
@@ -0,0 +1,46 @@
+from django.contrib import admin
+
+from .models import Poll, Choice
+
+
+class ChoiceInline(admin.TabularInline):
+    model = Choice
+    max_num = 2
+    extra = 2
+
+
+@admin.register(Poll)
+class PollAdmin(admin.ModelAdmin):
+    list_display = ('question', 'state', 'creation_date_time', 'publication_date_time', 'publication_end_date_time')
+
+    inlines = [
+        ChoiceInline
+    ]
+
+    fieldsets = [
+        (
+            'Question',
+            {
+                'fields': ['question']
+            }
+        ),
+        (
+            'Auteur',
+            {
+                'fields': ['user']
+            }
+        ),
+        (
+            'Validation',
```

Comment:
We never decided if the language of the administration panel should be French or English, but I think this is a reasonable choice to make it easier for new administrators.